include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -322,8 +366,20 @@ struct __parallel_scan_submitter<_CustomName, __internal::__optional_kernel_name
         __wgroup_size = ::std::min({__wgroup_size, __wgroup_size_kernel_1, __wgroup_size_kernel_2});
 #endif
 
-        // Practically this is the better value that was found
-        constexpr decltype(__wgroup_size) __iters_per_witem = 16;
+#ifdef _ONEDPL_SCAN_ITER_SIZE
+        constexpr decltype(__wgroup_size) __iters_per_witem = _ONEDPL_SCAN_ITER_SIZE;
+#else
+        constexpr decltype(__wgroup_size) __iters_per_witem = __iters_per_item(_AlgoType::value);
+#endif
+
+#ifdef _ONEDPL_SCAN_TUNING_MODE
+        // Override wgsize if performing a tuning sweep
+        if (const char* __wgroup_size_override = ::std::getenv("_ONEDPL_SCAN_WG_SIZE"))
```

Comment:
As far as I found, this is the first call of ```std::getenv``` in our code.
May be better to avoid it and make this logic somehow in```CMakeLists.txt``` file?

Reply:
It is an artifact from when I was testing with different work-group sizes. I don't see much much benefit from leaving it in the code, so I will remove it along with the other override macro for the iterations per work-item.